scrape/scrape_test.go

```diff
@@ -334,6 +341,15 @@ func TestScrapePoolReload(t *testing.T) {
 
 	require.Equal(t, sp.activeTargets, beforeTargets, "Reloading affected target states unexpectedly")
 	require.Len(t, sp.loops, numTargets, "Unexpected number of stopped loops after reload")
+
+	got, err := gatherLabels(reg, "prometheus_target_reload_length_seconds")
+	require.NoError(t, err)
+	testutil.RequireEqual(t, []labels.Labels{labels.FromStrings("interval", "3s")}, got["prometheus_target_reload_length_seconds"])
+	require.NoError(t, client_testutil.GatherAndCompare(reg, bytes.NewBufferString(`
```

Comment:
because it's a counter, I'd just go with `require.Equal(t, 1.0, client_testutil.ToFloat64(sp.metrics.targetScrapePoolReloads))`